languages/cpp/ExampleUse.md

```diff
@@ -1,21 +1,26 @@
-# EXAMPLES
+# Examples
```

Comment:
It's a bit confusing that we have 3 different `.md` file, it's hard to distinguish what this file is for compared to `CMakeBuild`.
How about joining the two together into one `INSTALL.md` ?
Or moving the `ExampleUse.md` into `examples` directory and then just mention it in the top MD file ?

Reply:
Went with moving `ExampleUse.md` into the examples' folder.

It's written from the perspective of how to consume the C++ SDK library with our `Wrapper.cpp` example.
https://github.com/bitwarden/sdk/pull/854/commits/8e5d5171e7249da78791d5d2546b1eb950aefc9b
https://github.com/bitwarden/sdk/pull/854/commits/34f582ad718e7ea6c0b76569123a978d6bff3b61


I had assumed that `CMakeBuild.md` was a standard with CMake for build documentation.
Looking online, it doesn't look like that is the case.